packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -490,6 +499,14 @@ class SliderThemeData with Diagnosticable {
   /// The color given to the [valueIndicatorShape] to draw itself with.
   final Color valueIndicatorColor;
 
+  /// The surface color that the slider is resting on.
```

Comment:
If this refers to the color of the slider's parent and/or the slider's overlapping (and lower z-order) sibling[s], then it doesn't seem like a good idea. We can't assume that the appearance of the component[s] behind the slider are rendered with just one color.

Reply:
@clocksmith

Reply:
See https://github.com/flutter/flutter/pull/52663#discussion_r398119384

Reply:
The surfaceColor can be removed. It was originally here because the disabled thumb color is translucent, but the opaque version of that color should be used if it were on a white background to match the material spec.

For cases when the developer explicitly sets the thumbColor to be translucent, the track can show beneath it as expected